plasmapy/formulary/parameters.py

```diff
@@ -1051,6 +1083,10 @@ def gyroradius(
     return r_Li
 
 
+rc = rhoc = gyroradius
```

Comment:
Same here, looks like this really doesn't handle multiple assignments, which is a shame...

Reply:
Ah. It is indeed. I'll fix it.

Reply:
Also done in f1aa1d4